### PR TITLE
Renames to bring in line with paper and interval-related fixes

### PR DESCRIPTION
**Pull request by mfherbst**

### CODE REVIEW

File: src/PlaneWaveBasis.jl [hunk 1]
```diff
@@ -135,7 +135,7 @@ build_kpoints(basis::PlaneWaveBasis, kcoords) =
     # operations are inverse to each other). The convention we want is
     # ψ(r) = sum_G c_G e^iGr / sqrt(Ω)
     # so that the ifft is normalized by 1/sqrt(Ω). It follows that the
-    # fft must be normalized by sqrt(Ω)/length
+    # fft must be normalized by sqrt(Ω/length)
```
Comment (antoine-levitt) [inline src/PlaneWaveBasis.jl:138]: ?

File: src/scf/mixing.jl [hunk 2]
```diff
@@ -136,15 +140,15 @@ end
         δρ = devec(x)
         Jδρ = copy(δρ)
 
-        # Apply Kernel (just vc for RPA and (vc + fxc) if not RPA)
+        # Apply Kernel (just vc for RPA and (vc + K_{xc}) if not RPA)
```
Comment (antoine-levitt) [inline src/scf/mixing.jl:143]: refer to the paper somewhere?

Comment (mfherbst) [reply src/scf/mixing.jl:143]: Right at the top?

Comment (antoine-levitt) [reply src/scf/mixing.jl:143]: Yeah for instance, or in the doc of hybrid (or both)

File: src/PlaneWaveBasis.jl (hunk 1 above)
Comment (mfherbst) [reply src/PlaneWaveBasis.jl:138]: ups ... not sure how that happened.

**Pull request merged**